edge-modules/DirectMethodSender/Settings.cs

```diff
@@ -25,15 +25,15 @@ public class Settings
                     configuration.GetValue<string>("TargetModuleId", "DirectMethodReceiver"),
                     configuration.GetValue<TransportType>("TransportType", TransportType.Amqp_Tcp_Only),
                     configuration.GetValue<TimeSpan>("DirectMethodDelay", TimeSpan.FromSeconds(5)),
-                    configuration.GetValue<Uri>("AnalyzerUrl", new Uri("http://analyzer:15000")));
+                    configuration.GetValue<string>("AnalyzerUrl", string.Empty));
```

Comment:
I didn't want to change this away from Uri type, but couldn't think of a way around this. Uri with empty string throws an exception.

Reply:
(Option.Maybe(configuration.GetValue<string>("AnalyzerUrl"))), then use Map method to convert it to Option<Uri>.